server/controllers/homepage.js

```diff
@@ -34,8 +34,8 @@ module.exports = function(app) {
       getTotalCollectives(),
       getTotalDonors(),
       queries.getTotalDonations(),
-      models.Group.getGroupsSummaryByTag('open source'),
-      models.Group.getGroupsSummaryByTag('meetup'),
+      models.Group.getGroupsSummaryByTag('open source', 3, [], 100, true),
+      models.Group.getGroupsSummaryByTag('meetup', 3, [], 100, true),
```

Comment:
I know but still, don't we want to show the top?

I'll merge it for now, we can try and see the results.